snappiershot/snapshot/snapshot.py

```diff
@@ -29,15 +29,18 @@ def __init__(self, configuration: Optional[Config] = None) -> None:
         self._metadata: Optional[SnapshotMetadata] = None
         self._snapshot_file: Optional[_SnapshotFile] = None
 
-    def assert_match(self, value: Any, exact: bool = False, update: bool = False) -> bool:
-        """ Assert that the given value matches the snapshot on file
+    def assert_match(
+        self, value: Any, exact: bool = False, update: bool = False, ignore: List[str] = []
+    ) -> bool:
+        """Assert that the given value matches the snapshot on file
 
         Args:
             value: new value to compare to snapshot
             exact: if True, enforce exact equality for floating point numbers,
               otherwise assert approximate equality
             update: if True, overwrite snapshot with given value
               (assertion will always pass in this case)
+            ignore: if set, will ignore variables names from the metadata
```

Comment:
i initially misread this as if `ignore` is a `set` type. suggest to remove 'if set, will ' and if desired, add `ignore specified variable names...`

Reply:
fixed [6a41b8e](https://github.com/MORSECorp/snappiershot/pull/96/commits/6a41b8e2f6cafa99f04da6517f7776b3ed16363f)